### DISCUSSION TITLE
What happens when I rename a GitHub team?

### DISCUSSION THREAD

**Discussion by markcellus** [community/community#21843] [New to GitHub]
<p>I would like to rename a GitHub team that has admin rights to quite a few repositories and is listed in a lot of CODEOWNERS files. If I rename the team, would this affect the admin rights in any way? How does this affect all of the places the old GitHub team is mentioned, like in the CODEOWNERS files?</p>

**Comment by seveas** [comment, marked as answer]
<p>I believe that the admin rights will be fine, I’ve seen quite some team renames and don’t remember ever losing access because of that.</p>
<p>However, all your CODEOWNERS files are going to need a manual update.</p>
